pkg/sql/compile/scope.go

```diff
@@ -456,13 +457,8 @@ func (s *Scope) Update(ts uint64, e engine.Engine) (uint64, error) {
 
 // Run read data from storage engine and run the instructions of scope.
 func (s *Scope) Run(e engine.Engine) (err error) {
```

Comment:
Recover @fengttt 's moerr handler here

Reply:
Note that I have adjusted the position of the moerr because Pipeline needs to do some clean-up operations when it goes wrong and cannot just exit.